src/api/tests/test_user.py

```diff
@@ -39,7 +39,9 @@ def test_user_post_success(post_user, client: TestClient, event_loop: asyncio.Ab
     '''
     add a valid new user into the session
     '''
+    r = client.post("/api/session", json=TEST_USER)
```

Comment:
This shouldn't be here, we are creating a new user so signing into any user here does not make sense.